extensions/cornerstone/src/utils/measurementServiceMappings/ArrowAnnotate.js

```diff
@@ -69,6 +69,7 @@ const Length = {
       metadata,
       referenceSeriesUID: SeriesInstanceUID,
       referenceStudyUID: StudyInstanceUID,
+      frameNumber: mappedAnnotations[0].frameNumber || 1,
```

Comment:
can't `mappedAnnotations` be undefined or `[]` which makes `mappedAnnotations[0]` undefined? so how about `mappedAnnotaions?[0]?.frameNumber`?

Reply:
done in fed74aae7ada884c3563b114ed8bc46938e325b3